test/unit/spec/EnrollCall_spec.js

```diff
@@ -46,7 +44,7 @@ function (Q, Okta, OktaAuth, LoginUtil, Util, Form, Beacon, Expect, $sandbox,
       router.on('afterError', afterRenderHandler);
       Util.registerRouter(router);
       Util.mockRouterNavigate(router, startRouter);
-      return tick()
+      return Q()
```

Comment:
I wonder if could we just return what is returned in the next `then` block? instead of `Q`
```
setNextResponse(resp || resAllFactors);
return Util.mockIntrospectResponse(router, resp || resAllFactors);
```

Reply:
relied in another similar comment.